### ISSUE TITLE
1.3.1 Info & Relationships - is F87 still relevant now that - IE is DEAD

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#2540]
CSS Pseudo before/after content has been supported now for several years and is part of the W3C AccName spec.  It is directly added to the accessibility tree within Chrome, Firefox, Edge, and Safari.

Now that IE11 is dead, what is the purpose of keeping "[Failure of Success Criterion 1.3.1 due to inserting non-decorative content by using ::before and ::after pseudo-elements and the 'content' property in CSS](https://www.w3.org/WAI/WCAG21/Techniques/failures/F87.html)" as a fail for 1.3.1 Info and Relationships.

**Comment by JAWS-test** [comment]
Only the old version of F87 was about output through the screen reader, which doesn't work in IE. In the [current version](https://www.w3.org/WAI/WCAG21/Techniques/failures/F87) it is only about the fact that the content may not be visible if I use custom user styles:

> Check that non-decorative information presented by the generated content is available when styles are overridden. 

This was discussed and decided in: https://github.com/w3c/wcag/issues/433
